src/viewer.js

```diff
@@ -140,42 +133,28 @@ async function generateReport(bundleStats, opts) {
   if (!chartData) return;
 
   await new Promise((resolve, reject) => {
-    ejs.renderFile(
-      `${projectRoot}/views/viewer.ejs`,
-      {
+    try {
```

Comment:
Looks like there is no need in promise here anymore. The code is sync

Reply:
It will be a breaking change.

Reply:
I mean the function can be still async but promise wrapper around sync code is not necessary.

Reply:
Ah, yep, agree. @realityking could you change it please?

Reply:
Pushed a commit to do this :)